docs/docker.md

```diff
@@ -0,0 +1,101 @@
+### Docker
+
+#### Building the image
+
+```
+docker build -t github/backup-utils .
+```
+
+#### Setting configuration options at runtime
+The `backup.config` file is dynamically populated at runtime with all `GHE_` environment variables that are part of the run command or Docker environment:
+
+```
+$ docker run -it -e "GHE_HOSTNAME=hostname" \
+-e "GHE_DATA_DIR=/data" \
+-e "GHE_EXTRA_SSH_OPTS=-i /ghe-ssh/id_rsa -o UserKnownHostsFile=/ghe-ssh/known_hosts \
+-e "GHE_NUM_SNAPSHOTS=15" \
+-v ghe-backup-data:/data \
```

Comment:
Should probably quote this to be safe as peeps are likely to copy & paste and then change inline.